l1-contracts/test/Decoder.t.sol

```diff
@@ -177,4 +195,21 @@ contract DecoderTest is Test {
       logsHash, referenceLogsHashFromIteration2, "Logs hash should be 0 when there are no logs"
     );
   }
+
+  function _populateInbox() internal returns (bytes32[] memory) {
```

Comment:
is a more suitable name for this recalculate inbox since its not actually using these values inside the test, more at the end just to check

Reply:
We're thinking about doing two separate functions, just seemed a bit "meh" as we would compute the same values so were just cleaner to use the same both places.